chipy_org/apps/sponsors/models.py

```diff
@@ -4,9 +4,10 @@
 
 
 class MeetingSponsor(models.Model):
-    sponsor = models.ForeignKey("sponsors.Sponsor")
-    meeting = models.ForeignKey("meetings.Meeting", related_name="meeting_sponsors")
-    about = models.TextField("About this sponsorship", blank=True, null=True)
+    sponsor = models.ForeignKey("sponsors.Sponsor", on_delete=models.PROTECT)
+    meeting = models.ForeignKey("meetings.Meeting", related_name="meeting_sponsors",
+            on_delete=models.CASCADE)
+    about = models.TextField(u"About this sponsorship", blank=True, null=True)
     about_short = models.CharField(
         "Brief description of sponsorship", max_length=128, blank=True, null=True
```

Comment:
After thinking about it for awhile, wouldn't the 'u' be unnecessary in the 'about' field since Python 3.0, the 'str' is by default represented as unicode characters. 

https://docs.python.org/3/howto/unicode.html#the-string-type
From that link:
> The String Type
> Since Python 3.0, the language’s str type contains Unicode characters, meaning any string created using "unicode rocks!", 'unicode rocks!', or the triple-quoted string syntax is stored as Unicode.

Reply:
That's a good catch.  Definitely not needed now that we are firmly in the world of python3.  We can remove those as we come across them.